extensions/wikia/UserLogin/UserLoginSpecialController.class.php

```diff
@@ -31,29 +31,28 @@ public function __construct() {
 	public function init() {
 		$loginTitle = SpecialPage::getTitleFor( 'UserLogin' );
 		$this->formPostAction = $loginTitle->getLocalUrl();
-		$this->isMonobookOrUncyclo = ( $this->wg->User->getSkin() instanceof SkinMonoBook || $this->wg->User->getSkin() instanceof SkinUncyclopedia );
+		$this->isMonobookOrUncyclo = ( $this->wg->User->getSkin() instanceof SkinMonoBook ||
+			$this->wg->User->getSkin() instanceof SkinUncyclopedia );
 		$this->userLoginHelper = (new UserLoginHelper);
 	}
 
 	private function initializeTemplate() {
-		//Oasis/Monobook, will be filtered in AssetsManager :)
-		$this->response->addAsset( 'extensions/wikia/UserLogin/css/UserLogin.scss' );
-
-		if ( !empty( $this->wg->EnableFacebookClientExt ) ) {
+		// Load Facebook JS if extension is enabled and skin supports it
+		if ( !empty( $this->wg->EnableFacebookClientExt ) && !$this->isMonobookOrUncyclo ) {
 			$this->response->addAsset( 'extensions/wikia/UserLogin/js/UserLoginFacebookPageInit.js' );
-			$this->response->addAsset( 'extensions/wikia/UserLogin/js/FacebookLogin.js' );
```

Comment:
Correct. It's now being loaded on every logged out page view https://github.com/Wikia/app/blob/uconn-5/extensions/wikia/AssetsManager/config.php#L409, and it's not needed for logged in users.